diagnostic.md

```diff
@@ -57,13 +67,15 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$('li:parent^2').hide();
```

Comment:
This is not a valid selector.  One possible way is `$('li').parent().parent().hide();`.